src/docfx/Models/PdfJsonConfig.cs

```diff
@@ -29,6 +29,9 @@ public class PdfJsonConfig : BuildJsonConfig
         [JsonProperty("keepRawFiles")]
         public bool KeepRawFiles { get; set; }
 
+        [JsonProperty("includeDefaultToc")]
+        public bool IncludeDefaultToc { get; set; }
```

Comment:
To keep backward compatibility, set default value of this to `true`? The default value is used if neither `docfx.json` nor CLI options sets it.

Or, you can also change it to `ExcludeDefaultToc`, the we need not to explicitly set the default value as it is `false` by default.